File: backend/src/main/java/com/ssafy/happynurse/domain/reminder/dto/PreviewSttReminderResponse.java
package com.ssafy.happynurse.domain.reminder.dto;

import io.swagger.v3.oas.annotations.media.Schema;

@Schema(description = "STT 발화 시간 파싱 결과")
public record PreviewSttReminderResponse(
        @Schema(description = "파싱된 알람 시각 (epoch millis, KST)") long fireAtEpochMillis
) {
}

File: frontend/mobile/wear/src/main/java/com/happynurse/wear/data/remote/model/PreviewSttReminderResponse.kt
// /reminders/stt/preview 응답 — 파싱된 알람 시각(epoch millis, KST).
package com.happynurse.wear.data.remote.model

import kotlinx.serialization.Serializable

@Serializable
data class PreviewSttReminderResponse(
    val fireAtEpochMillis: Long? = null,
)
